src/components/NcChip/NcChip.vue

```diff
@@ -70,7 +71,7 @@ export default {
 
 <template>
 	<div class="nc-chip" :class="{ [`nc-chip--${type}`]: true, 'nc-chip--no-actions': noClose && !hasActions() }">
-		<span class="nc-chip__icon">
+		<span class="nc-chip__icon" :class="{ 'nc-chip__icon--empty': !hasIcon() }">
```

Comment:
Why not `v-if="hasIcon()"` to not render it at all?

Reply:
well that seems to be cleaner, adjusted